wiki/Shortcut_key_reference/nl.md

```diff
@@ -0,0 +1,290 @@
+| `F1` | Switcht naar de [compose](/wiki/Compose) modus. |
+| `F2` | Switcht naar de [design](/wiki/Design) modus. |
+| `F3` | Switcht naar de [timing](/wiki/Timing) modus. |
+| `F4` | Opent de [song setup](/wiki/Song_Setup) dialoog. |
+| `F5` | Opent de test modus. |
+| `F6` | Opent de timing en control points dialoog. |
+| `Ctrl` + `P` | Voegt een onafhankelijk timing punt op de huidige tijdspositie. Als je niet in de timing mode zit, opent dit ook de timing en control points dialoog. |
+| `Ctrl` + `Shift` + `P` | Voegt een afhankelijk timing punt toe op de huidige tijdspositie. Dit zal ook de timing en control points dialoog openen. |
+| `Ctrl` + `I` | Verwijdert de huidige (on)afhankelijke sectie. Dit werkt niet op het eerste onafhankelijke timing punt. |
+
+#### Muis
+
+| Sneltoets | Actie |
+| :-- | :-- |
+| `Mouse Wheel Up/Down` | Zoekt voor/achteruit. |
+| `Shift` + `Mouse Wheel Up/Down` | Zoekt voor/achteruit met een snelheid van 4x. |
+| `Ctrl` + `Mouse Wheel Up/Down` | Past de snap divisor aan. |
+| `Alt` + `Mouse Wheel Up/Down` | Past de distance snapping multiplier aan, wanneer je over het speelveld hangt. Past de timeline zoom aan als je over de timeline hangt. |
+| `Alt` + `Shift` + `Mouse Wheel Up/Down` | Past de distance snapping multiplier preciezer aan, wanneer je over het speelveld hangt. |
+
+### Compose
+
+#### Muis
+
+| Sneltoets | Actie |
+| :-- | :-- |
+| `Right Click` | Selecteert/deselecteert een nieuwe combo (wanneer je objecten plaatst) of verwijdert een object (als je aan het selecteren bent). |
+
+#### Algemeen
+
+| Sneltoets | Actie |
+| :-- | :-- |
+| `Alt` + `F4` of `Esc` | Keer terug naar de liedjes-selectie |
+| `Ctrl` + `Z` | Ongedaan maken |
+| `Ctrl` + `Y` | Opnieuw doen |
+| `Ctrl` + `A` | Selecteert alle hit-objecten. |
+| `Ctrl` + `X` | Knip. |
+| `Ctrl` + `C` | Kopieer. |
+| `Ctrl` + `V` | Plak. |
+| `Ctrl` + `D` | Dupliceert de selectie. Dit plakt de selectie 1 maat achter het laatst geselecteerde object. |
+| `Delete` | Verwijder. |
+| `1`, `2`, `3`, of `4` | Switcht tussen plaats/selectie modus: select, circle, slider en spinner (osu!standard, osu!taiko of osu!catch). |
+| `1`, `2`, of `3` | Switcht tussen plaats/selectie modus: select, circle en hold (osu!mania). |
+| `Ctrl` + `Alt` + `Mouse Wheel Up/Down` | Switch tussen plaats/selectie modus. |
+| `Ctrl` + `Shift` + `A` | Opent [AiMod](/wiki/Beatmap_Editor/AiMod). |
+
+#### Speel veld
+
+| Sneltoets | Actie |
+| :-- | :-- |
+| `Alt` | Toont de [distance snap](/wiki/Beatmap_Editor/Distance_Snap) zolang de toets is ingedrukt. |
+| `Y` | Toont/verbergt distance snap. |
+| `Shift` | Toont de grid snap zolang de toets is ingedrukt. |
+| `T` | Toont/verbergt de grid snap. |
+| `G` | Cirkel door de grid size. |
+| `Ctrl` + `1` | Zet de grid size naar tiny. |
+| `Ctrl` + `2` | Zet de grid size naar small. |
+| `Ctrl` + `3` | Zet de grid size naar medium. |
+| `Ctrl` + `4` | Zet de grid size naar large. |
```

Comment:
'Zet de grid size op tiny' of' Verandert de gridsize naar tiny' 4x

Grid size kan je eventueel veranderen naar rastergrootte

Reply:
Ik denk dat 'Grid Size' wel goed is aangezien 'Grid snap' ook wordt gebruikt en deze beide meer termen zijn.